A/Atomsk/bundled/patches/atomsk_64.patch

```diff
@@ -0,0 +1,101 @@
+diff --git a/src/Makefile b/src/Makefile
+index 9eb21e9..658445a 100644
+--- a/src/Makefile
++++ b/src/Makefile
+@@ -65,9 +65,11 @@ ifndef FORTRAN
+   # -I is used to specify folders where the compiler should look for *.mod files
+   # -J is used to specify the folder where the compiler will store the *.mod files
+   FFLAGS=-O2 $(OPENMP) -fno-backslash -I..$(SEP)$(OBJ) -J..$(SEP)$(OBJ)
++  EXEFFLAGS=-O2 $(OPENMP) -fno-backslash -I$(OBJ) -J$(OBJ)
+   
+   # Add -fpic to compile Atomsk as a shared library
+-  #FFLAGS+=-fPIC
++  FFLAGS+=-fPIC
++  EXEFFLAGS+=-fPIC
+   
+   # Un-comment the following line to use 8-bits integers by default
+   # (number of atoms up to 9x10^18 instead of 2x10^9)
+@@ -157,8 +159,8 @@ all: atomsk
+ 
+ # Compile the program
+ # Note that the preprocessor must be called (-cpp)
+-atomsk: message modes compute general
+-	$(FORTRAN) $(FFLAGS) -cpp -o $(BIN) $(OBJ)$(SEP)*.o atomsk.f90 -I $(OBJ) $(LAPACK)
++atomsk: lib
++	$(FORTRAN) $(EXEFFLAGS) -cpp -o $(bindir)/atomsk$(exeext) atomsk.f90 -latomsk
+ 	@echo ""
+ 	@echo "    \o/ Compilation was successful!"
+ 	@echo ""
+@@ -169,7 +171,7 @@ atomsk: message modes compute general
+ # Compile the program as a library
+ # Note that the preprocessor must be called (-cpp)
+ lib: message compute modes
+-	$(FORTRAN) $(FFLAGS) -cpp -shared -o atomsk.so $(OBJ)$(SEP)*.o
++	$(FORTRAN) $(EXEFFLAGS) -cpp -shared -o $(libdir)/libatomsk.$(dlext) $(OBJ)$(SEP)*.o $(LAPACK)
+ 	@echo ""
+ 	@echo "    \o/ Atomsk was compiled as a shared library file, 'atomsk.so'."
+ 	@echo ""
+diff --git a/src/compute/compute_G.f90 b/src/compute/compute_G.f90
```

Comment:
It'd be easier to do this with some macros, but the build system doesn't seem to be sensible enough (they ask you to _edit_ the makefile....) to allow that.

Reply:
Yea, the package is just maintained by one person, and I think he's good with the way it is now.

Reply:
It isn't about preference, it's about one thing is needed or not: https://docs.binarybuilder.org/stable/building/#Expanding-C-string-ABIs-or-libgfortran-versions.  Here you have to expand the libgfortran version because we're linking to libgfortran, this code seems to be written entirely in Fortran so expanding the C++ string ABI makes no sense.